### PR TITLE
Tutorial redesign

### PR DESCRIPTION
**Pull request by H3rby7**
up to tutorial 5
still things to be done
branch getting very fat, so merge for the first parts necessary

### CODE REVIEW

File: elements/blockly/blocks.js [hunk 1]
```diff
@@ -90,10 +90,8 @@ Blockly.Blocks.move = {
       .appendField('Move')
       .appendField(
         new Blockly.FieldDropdown([
-          ['⇩ Forward ', 'FORWARD'],
-          ['⇦ Left', 'LEFT'],
-          ['⇧ Backward', 'BACKWARD'],
-          ['⇨ Right', 'RIGHT']
+          ['⇧ Forward ', 'FORWARD'],
+          ['⇩ Backward', 'BACKWARD']
```
Comment (timgrossmann) [inline elements/blockly/blocks.js:94]: Why did you remove moving left and right ?

**Review by timgrossmann — COMMENTED** [review]
I really don't know why, but blockly/the toolbox and the workspace don't work for me...
Not even a error message

screen shot 2017-02-20 at 21 33 21

File: assets/tutorial/01-blockly.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+const entities = require('../../models/game/entities')
```
Comment (paulsonnentag) [inline assets/tutorial/01-blockly.js:1]: For me the levels don't really feel like assets because you can execute arbitrary JavaScript here. You also have dependencies on other modules. That's why I originally put them into the tutorial module.

**Comment by H3rby7** [comment]
rebasing locally to continue work
